core/server/data/migrations/versions/4.0/08-add-members-login-events-table.js

```diff
@@ -0,0 +1,6 @@
+const {addTable} = require('../../utils');
+module.exports = addTable('members_login_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+    created_at: {type: 'dateTime', nullable: false}
```

Comment:
Just wondering if it makes sense to have a `source` here which can have the token/link that was used to login ? No strong usecase except just storing extra info for/when needed

Reply:
Hmm, I think we should add stuff as/when we have the usecase for it, at this stage. Wondering what that could be used for?

Reply:
> I think we should add stuff as/when we have the usecase for it, at this stage

Cool yea, definitely no immediate usecase in mind except any audit purposes, happy to leave it out :)

Reply:
imo there'd probably be a stronger use-case for geolocation or IP here 🤔